web/vtadmin/src/proto/vtadmin.js

```diff
@@ -35887,6 +35932,7 @@ $root.tabletmanagerdata = (function() {
          * Properties of a StartReplicationRequest.
          * @memberof tabletmanagerdata
          * @interface IStartReplicationRequest
+         * @property {boolean|null} [semiSync] StartReplicationRequest semiSync
```

Comment:
@notfelineit do these changes affect the "destructive" tablet actions UI?

Reply:
I don't think so, not as it exists - should we add a semiSync/durability option in the UI?

Reply:
I don't think so, not as it exists - should we add a semiSync/durability option in the UI?

Reply:
<img width="1284" alt="Screen Shot 2022-01-27 at 9 45 23 AM" src="https://user-images.githubusercontent.com/31225471/151414477-47620312-6a15-4425-926b-9b5bb7c0a642.png">
Yes! It follows vtctld2's ui in that there is a singular button

Reply:
I suspect that this directly calls the TabletManager grpc API instead of going through vtctld API.
My suggestion is to add a new vtctld API. Then semi_sync won't need to be provided by the user, it can be computed by VtctldServer.

Reply:
I'll check today and see which it is using - and switch to vtctld if necesary.

Reply:
@deepthi vtadmin uses this codepath: https://cs.github.com/vitessio/vitess/blob/f906693c6441f4b41d8f842ffb9dba85258b5158/go/vt/vtctl/grpcvtctldserver/server.go?q=StartReplication#L2439 it does call the tablet manager API, but I think this is the vtctld API mentioned?